mos-platform/nes-unrom-512/bank.s

```diff
@@ -18,7 +18,7 @@ banked_call:
 	sta __rc17 ; __rc17 = requested PRG bank
```

Comment:
Ah, I hadn't noticed that the RC value is preserved across the call, right.

Since `__rc20` is caller-saved, the value on entry to `banked_call` needs to be saved and restored too, a la `lda __rc20; pha`.

Reply:
Sorry! Should be fixed.

Reply:
Aaand this clobbers the value on entry in A; the store to `__rc17` needs to happen first ;)